ratelimits/names.go

```diff
@@ -85,9 +88,10 @@ var nameToString = map[Name]string{
 	NewRegistrationsPerIPv6Range:            "NewRegistrationsPerIPv6Range",
 	NewOrdersPerAccount:                     "NewOrdersPerAccount",
 	FailedAuthorizationsPerDomainPerAccount: "FailedAuthorizationsPerDomainPerAccount",
-	CertificatesPerDomain:                   "CertificatesPerDomain",
+    CertificatesPerDomain:                   "CertificatesPerDomain",
```

Comment:
nit: looks like your editor is doing something weird with indentation. Make sure your editor runs `gofmt` every time you save a file.